src/components/Movies.jsx

```diff
@@ -0,0 +1,22 @@
+import Movie from './Movie'
+import '../styles/movies.scss'
+
+const Movies = ({ movies, viewTrailer, closeCard }) => {
+
```

Comment:
The logic for implementing infinite scroll with a useInfiniteScroll hook using measureRef is missing.